x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -60,6 +60,10 @@ import { isUnrecoverableError } from './errors';
 const defaultBackoffPerFailure = 5 * 60 * 1000;
 export const EMPTY_RUN_RESULT: SuccessfulRunResult = { state: {} };
 
+export const TASK_MANAGER_TRANSACTION_TYPE = 'task-manager';
```

Comment:
The goal is to have 

| transaction.type  | transaction.name | labels.task
| ------------- | ------------- | ------------- | 
| task-manager  | run  | task 1 |
| task-manager  | mark-task-as-running  | task 1 |
| task-manager  | mark-task-as-running  | task 2 |

Reply:
Is this desired data convention documented somewhere? So that if we want to ever add a new transaction, we'll know what convention to follow?